packages/react-router/docs/Switch.md

```diff
@@ -51,4 +51,17 @@ This is also useful for animated transitions since the matched `<Route>` is rend
 
 ## children: node
 
-All children of a `<Switch>` should be `<Route>` elements. Only the first child to match the current location will be rendered.
+All children of a `<Switch>` should be `<Route>` or `<Redirect>` elements. Only the first child to match the current location will be rendered.
+
+`<Route>` elements are matched using their `path` prop and `<Redirect>` elements are matched using their `from` prop. A `<Route>` with no `path` prop or a `<Redirect>` with no `from` prop will always match the current location.
```

Comment:
I think this is not exactly right - both `from` and `path` are accepted (`path` has the priority) for both <Route> and <Redirect> according to this code: https://github.com/ReactTraining/react-router/pull/4546/files/bf4a4ca251807f4fd054b0d6797c567d7da8140b#diff-684a00184c10908924cecd2101b7dd79